opencog/benchmark/benchmark.py

```diff
@@ -7,12 +7,9 @@
 Displays the number of operations per second achieved under various
 scenarios utilizing the Python bindings that wrap the AtomSpace using
 Cython. Some of the tests compare the evaluation of nodes using the
-Python Scheme bindings which take much longer than their comparable
-Cython bindings. XXX Which is utterly false and insane -- the
-measurements are being done incorrectly, since the Scheme bindings
-normally run about 3x to 10x faster than the Python bindings do.
-Python is generally quite slow, and is very unlikely to be faster than
-scheme; something is borked with this test case.
+Python scheme_eval_h which takes much longer than the comparable
+Cython bindings because of the overhead of creating and initializing
+a SchemeEval object.
```

Comment:
Because this is a benchmark timing Python bindings and execution. The statement says that calling Python bindings compared to scheme_eval_h takes longer than the comparable Cython bindings. That is most definitely true.

There is a test which compare the Python bindlink against a call of cog-bind using scheme_eval_h.

This is not a statement about the relative speed of Python versus scheme at all. It is a comment about the speed of the scheme binding as currently implemented.

At any rate, I am working on fixing this so that the scheme binding doesn't take so much time using the suggestions you made in the request that Jim Rutt made on the email list. So this comment is short lived anyway. I'm going to take it away next week after the overhead is limited.